cluster_agent/utils/exception.py

```diff
@@ -24,3 +38,85 @@ class JobbergateApiError(ClusterAgentError):
 
 class JobSubmissionError(ClusterAgentError):
     """Raise exception when a job cannot be submitted raises any error"""
+
+
+class SlurmParameterParserError(ClusterAgentError):
+    """Raise exception when Slurm mapper or SBATCH parser face any error"""
+
+
+@contextlib.asynccontextmanager
+async def handle_errors_async(
```

Comment:
I would welcome this as a PR on py-buzz. Adding some support for async would be good!

Reply:
Nice, I'll keep it in mind and open the PR when I have the chance.